src/components/modals/SecurityAlertsModal.tsx

```diff
@@ -19,56 +23,44 @@ interface OwnProps {
 }
 type Props = OwnProps & ThemeProps
 
-class SecurityAlertsModalComponent extends React.Component<Props> {
-  render() {
-    const { bridge } = this.props
+const SecurityAlertsModalComponent = ({
+  bridge,
+  messages,
+  selectUser
+}: Props) => {
```

Comment:
Please do `(props: Props)` as the param list, and then do `const { ... } = props` inside the component. We do this pretty consistently throughout the GUI, and after my refactoring work, the login UI does it pretty consistently too now.